woodwork/table_accessor.py

```diff
@@ -379,7 +275,7 @@ def mutual_information(self, num_bins=10, nrows=None):
             Mutual information values are between 0 (no mutual information) and 1
             (perfect dependency).
         """
-        mutual_info = self.mutual_information_dict(num_bins, nrows)
+        mutual_info = _get_mutual_information_dict(self._dataframe, num_bins=num_bins, nrows=nrows)
```

Comment:
Can this call remain the same since `self.mutual_information_dict` is still a valid method? Doesn't matter too much, but the original call looks slightly cleaner.

Reply:
Good call. Changing